jobs/JRAP_PROCESS_HYDRO

```diff
@@ -86,31 +86,27 @@ setpdy.sh
 ##############################################
 
 # for testing with canned data
-export DCOMROOT=/lfs/h1/ops/$envir/dcom
-export COMROOT=${COMROOT:-/lfs/h2/emc/ptmp/Ming.Hu/com}
-export COMIN=${COMIN:-${COMROOT}/${NET}/${rap_ver}/${RUN}.${PDY}}
-export COMOUT=${COMOUT:-${COMROOT}/${NET}/${rap_ver}/${RUN}.${PDY}}
+#export DCOMROOT=/lfs/h1/ops/$envir/dcom
+#export COMROOT=${COMROOT:-/lfs/h2/emc/ptmp/Ming.Hu/com}
+#export COMIN=${COMIN:-${COMROOT}/${NET}/${rap_ver}/${RUN}.${PDY}}
+#export COMOUT=${COMOUT:-${COMROOT}/${NET}/${rap_ver}/${RUN}.${PDY}}
 
+# for wcoss2 ops
+export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${rap_ver}/${RUN}.${PDY})}
+export COMOUT=${COMOUT:-$(compath.py ${envir}/${NET}/${rap_ver}/${RUN}.${PDY})}
 export COMINradar=${COMINradar:-$(compath.py hourly/prod)}
 export COMINmosaic=${COMINmosaic:-${DCOMROOT}/ldmdata/obs/upperair/mrms}
 export PREPDIR=${PREPDIR:-$(compath.py ${envir}/${NET}/${rap_ver})}
 
-# for wcoss2 ops
-#export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${rap_ver}/${RUN}.${PDY})}
-#export COMOUT=${COMOUT:-$(compath.py ${envir}/${NET}/${rap_ver}/${RUN}.${PDY})}
-#export COMINradar=${COMINradar:-$(compath.py hourly/prod)}
```

Comment:
For COMINradar, this is the correct path for WCOSS2:
export COMINradar=${COMINradar:-$(compath.py $envir/hourly/$radar_ver)}

I set radar_ver=v0.0 in versions/run.ver.  But this isn't a big deal since COMINradar is only used if the script can't find the mosaic data.